### ISSUE TITLE
GPXSee-maps/World/Europe/ES/ don't work for me

### ISSUE THREAD

**Issue by jacanestro** [tumic0/GPXSee-maps#47]
Hi!!

First of all, I'd like to say that this is a great program! Congratulations!

I'm not an expert, so sorry if this is a dummy question.

The Spanish IGN raster maps don't work for me.
I found in the Spanish IGN web site this link: https://www.ign.es/wms-inspire/mapa-raster
which redirects you to: https://www.ign.es/wms-inspire/mapa-raster?request=GetCapabilities&service=WMS

But I don't know how to handle it and what to do to update GPXSee IGN-mapa-raster.xml file.
I have replace the url line with "<url>https://www.ign.es/wms-inspire/mapa-raster</url>" but it does not work.

Can you help me with this issue???

Thank you in advance!

**Comment by sikmir** [comment]
@jacanestro I've just tried [IGN-mapa-raster.xml](https://github.com/tumic0/GPXSee-maps/blob/master/World/Europe/ES/IGN-mapa-raster.xml) and it works for me.

**Comment by jacanestro** [comment]
Thank you! I have reinstalled GPXSee and map XML without success.
I'm using Windows 10. Where can I see the console output?

**Comment by tumic0** [comment]
The problem is, the map zooms use the colon (":") charcter which is not allowed on Windows to be part of a path (GPXSee uses the zoom names in the tile file names). I have changed the map to use the EPSG:3857 matrix rather than the EPSG:4326 (no idea why I chose that one when creating the map as it is much more distorted than the EPSG:3857 one) which has a more sane zoom names. So the map now works on Windows (and looks "better"), but it has to be fixed in GPXSee anyway.
